### PR TITLE
doc/rados: update prompts in crush-map-edits.rst

### PR DESCRIPTION
**Pull request by zdover23**
This PR updates the prompts in crush-map-edits.rst to make them unselectable.

There remains no good known way to render parts of a file if the first line of that file begins with a hash (#). Sphinx italicizes such a first line, which is not what I want. Two examples of this are present in the file crush-maps-rst under the section called "CRUSH Map Bucket Types". I set this down here for my own records, in case it is helpful in finding a way to make these sections render as I would prefer.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by anthonyeleven — CHANGES_REQUESTED** [review]
LGTM, modulo spelling of “prompts”

**Comment by zdover23** [comment]
> LGTM, modulo spelling of “prompts”

Yeez.

File: doc/rados/operations/crush-map-edits.rst
Comment (colemitchell) [inline doc/rados/operations/crush-map-edits.rst:668]: @zdover23 This sentence is garbled.

File: doc/rados/operations/crush-map-edits.rst
Comment (colemitchell) [inline doc/rados/operations/crush-map-edits.rst:656]: @zdover23 The _to_ in "to the CRUSH map" should be _against_. And perhaps "comes back out" should be replaced with "is output". There is also an oddity involving the double use of _ensure_.

Comment (anthonyeleven) [reply doc/rados/operations/crush-map-edits.rst:656]: Agreed on both counts.  Suggest one instance of "ensure" to "check", probably the second.

File: doc/rados/operations/crush-map-edits.rst
Comment (colemitchell) [inline doc/rados/operations/crush-map-edits.rst:644]: s/comprised of/comprising/

Comment (anthonyeleven) [reply doc/rados/operations/crush-map-edits.rst:644]: My sense is that both are acceptable.  "comprise" is so protean and slippery that I personally usually find a different phrasing to avoid being stilted. ymmv.

File: doc/rados/operations/crush-map-edits.rst
Comment (colemitchell) [inline doc/rados/operations/crush-map-edits.rst:172]: @zdover23 I would remove the _etc._

Comment (anthonyeleven) [reply doc/rados/operations/crush-map-edits.rst:172]: Agreed, with `e.g.` in there it's redundant.

File: doc/rados/operations/crush-map-edits.rst
Comment (zdover23) [reply doc/rados/operations/crush-map-edits.rst:668]: @colemitchell, no kidding. Much of this file could stand a refinement pass. I will create a pull request that addresses your comments in this PR (and I appreciate your grammar and usage notes, but this is a prompt-updating PR, so I won't be scope creeping it).

Comment (zdover23) [reply doc/rados/operations/crush-map-edits.rst:668]: @colemitchell: I have made all the corrections that you have suggested in this PR: https://github.com/ceph/ceph/pull/48362

Would you check the prompts in this PR, make sure that I have indeed rendered them unselectable, and then give me an approval? (Then, if Anthony hasn't beaten you to it, check out https://github.com/ceph/ceph/pull/48362.)

Comment (zdover23) [reply doc/rados/operations/crush-map-edits.rst:668]: @colemitchell: This PR should probably be merged before https://github.com/ceph/ceph/pull/48362, because I cherry-picked the commit in this PR into the branch I created for that PR. I won't bore you with the details, but it would really reduce the amount of unnecessary work that I'll have to do if you approve this one prior to approving that one.

File: doc/rados/operations/crush-map-edits.rst
Comment (zdover23) [reply doc/rados/operations/crush-map-edits.rst:644]: I have fixed this in https://github.com/ceph/ceph/pull/48362.

File: doc/rados/operations/crush-map-edits.rst
Comment (zdover23) [reply doc/rados/operations/crush-map-edits.rst:172]: I have fixed this in https://github.com/ceph/ceph/pull/48362.

File: doc/rados/operations/crush-map-edits.rst
Comment (zdover23) [reply doc/rados/operations/crush-map-edits.rst:656]: I have fixed this in https://github.com/ceph/ceph/pull/48362.

**Comment by zdover23** [comment]
@anthonyeleven: I can't merge this one until you give your approval. Let me know if this PR passes muster.

**Review by anthonyeleven — APPROVED** [review]

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/48363 - Quincy backport
https://github.com/ceph/ceph/pull/48364 - Pacific backport

**Comment by anthonyeleven** [comment]
For anyone stumbling across this PR, here's my understanding of what Zac's accomplishing here.

When we have example commands in a code block, a reader who cuts/pastes should only get the commands, not the shell prompt characters.

**Comment by zdover23** [comment]
> For anyone stumbling across this PR, here's my understanding of what Zac's accomplishing here.
> 
> When we have example commands in a code block, a reader who cuts/pastes should only get the commands, not the shell prompt characters.

This is correct.

**Pull request merged**
